src/i18n/fi/translation.json

```diff
@@ -151,6 +151,37 @@
   "interests": "Kiinnostukset",
   "interests-from-profile": "Osaamisprofiilista tuotu kiinnostukseni",
   "interests-of-your-choice": "Valitut kiinnostuksen kohteet",
+  "introduction": {
+    "step-1": {
+      "item-1": "Tunnista vahvuutesi ja kiinnostuksen kohteesi. Kaikki osaamiseesi ja kiinnostuksiisi liittyvät tiedot ovat nähtävillä yhdessä paikassa.",
+      "item-2": "Tallenna työ- ja koulutushistoriasi. Voit tuoda tietojasi myös muista palveluista, kuten Työmarkkinatorista ja Oma Opintopolusta.",
+      "item-3": "Löydä profiilisi perusteella työ- ja koulutusmahdollisuuksia, jotka sopivat juuri sinulle.",
+      "item-4": "Tallenna kiinnostavimmat ammatit sekä työ- ja koulutusvaihtoehdot suosikeiksi.",
+      "item-5": "Suunnittele koulutus- ja urapolkuasi, sekä seuraa edistymistäsi asettamasi päämäärän saavuttamiseen",
+      "text-1": "Kirjautuneena saat käyttöösi henkilökohtaisen osaamispolun, joka auttaa sinua hallinnoimaan ja hyödyntämään osaamistasi tehokkaasti.",
+      "text-2": "Teksin paikka...",
+      "title": "1. Tervetuloa luomaan Oma Osaamispolkusi!"
+    },
+    "step-2": {
+      "item-1": "Käytämme tietojasi vain anonymisoituna, jotta voimme luoda yleiskuvan käyttäjien tarpeista.",
+      "item-2": "Älä syötä henkilötietojasi tai arkaluontoisia tietoja palvelun tekstikenttiin.",
+      "item-3": "Tietojenkäsittelymme perustuu Lakiin valtakunnallisesta jatkuvan oppimisen osaamistietovarannosta. Lisätietoja tietosuojaselosteestamme.",
+      "item-4": "Voit itse määritellä, miten ja missä tietojasi käytetään asetuksissa.",
+      "text-1": "Palvelun käyttö on vapaaehtoista ja turvallista. Käsittelemme tietojasi luottamuksellisesti ja lainmukaisesti.",
+      "text-2": "Teksin paikka...",
```

Comment:
Typo.. but on the other hand it seems to be "TODO"/placeholder text altogether :D